gu-model/src/envman.rs

```diff
@@ -108,13 +108,21 @@ impl Default for ResourceFormat {
     }
 }
 
+#[derive(Clone, Serialize, Deserialize, Default, Hash, Eq, PartialEq, Debug)]
+#[serde(rename_all = "camelCase")]
+pub struct ExecOptions {
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub working_dir: Option<String>,
+}
+
 #[derive(Clone, Serialize, Deserialize, Hash, Eq, PartialEq, Debug)]
 #[serde(rename_all = "camelCase")]
 pub enum Command {
     Exec {
         // return cmd output
         executable: String,
         args: Vec<String>,
+        options: ExecOptions,
```

Comment:
Change is not backward compatible. it breaks current applications. #[serde(default)] is minimal